diff --git a/apps/api/src/common/database/prisma.service.ts b/apps/api/src/common/database/prisma.service.ts
index 9702e03..a0e8c40 100644
--- a/apps/api/src/common/database/prisma.service.ts
+++ b/apps/api/src/common/database/prisma.service.ts
@@ -1,11 +1,5 @@
 import { PrismaPg } from '@prisma/adapter-pg';
-import {
-  Inject,
-  Injectable,
-  Logger,
-  type OnModuleDestroy,
-  type OnModuleInit
-} from '@nestjs/common';
+import { Inject, Injectable, Logger, type OnModuleDestroy, type OnModuleInit } from '@nestjs/common';
 import { AppConfigService } from '../../config/app-config.service';
 import { PrismaClient } from '../../generated/prisma/client';
 
